lib/runner.js

```diff
@@ -203,16 +209,21 @@ Runner.prototype.run = function () {
 			}
 		})
 		.then(function () {
-			return self.serial(tests.serial);
+			return tests.only.length ? self.concurrent(tests.only) : [];
```

Comment:
it can be just `return self.concurrent(tests.only)`, since `tests.only` defaults to `[]`.